.flowconfig

```diff
@@ -0,0 +1,7 @@
+[version]
+0.53.0
+
+[ignore]
+<PROJECT_ROOT>/node_modules/react-element-to-jsx-string
```

Comment:
I'm surprised we need to comment things out in a completely clean project! Is this normal?

Reply:
Some node modules have `// @flow` files with syntax that our version doesn't handle or intentionally malformed syntax for testing purposes.  It would be nice if flow would look at other .flowconfigs and just bail if that's the case (or bail if there is no .flowconfig for a npm module with `// @flow` files).

Reply:
Blah, ok!